server/service_test.go

```diff
@@ -125,6 +126,41 @@ func blindedBlockContentsToPayloadDeneb(signedBlindedBlockContents *eth2ApiV1Den
 	}
 }
 
+func blindedBlockContentsToPayloadElectra(signedBlindedBlockContents *eth2ApiV1Electra.SignedBlindedBeaconBlock) *builderApiDeneb.ExecutionPayloadAndBlobsBundle {
```

Comment:
shouldn't this return `*builderApiElectra` instead of `builderApiDeneb`?

Reply:
Nah because there is no new `ExecutionPayloadAndBlobsBundle` in Electra.